lib/auth/auth_login_test.go

```diff
@@ -815,6 +815,169 @@ func TestCreateRegisterChallenge_unusableDevice(t *testing.T) {
 // -----END PRIVATE KEY-----
 const sshPubKey = `ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBGv+gN2C23P08ieJRA9gU/Ik4bsOh3Kw193UYscJDw41mATj+Kqyf45Rmj8F8rs3i7mYKRXXu1IjNRBzNgpXxqc=`
 
+func TestServer_AuthenticateUser_passwordOnly(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	svr := newTestTLSServer(t)
```

Comment:
Suggestion/nit: Use `testServer`? It's less ambiguous and more readable. Same throughout.

Reply:
It's really funny how I'm falling into the same traps I'm constantly complaining about. :D

Reply:
`srv` and `svr` are a particular pet peeve of mine, I just really dislike those. That said, "server" can be pretty confusing in some context, as we have so many things that can be called a server.